api/src/test/java/org/openmrs/validator/ProviderAttributeTypeValidatorTest.java

```diff
@@ -40,21 +57,51 @@ public void validate_shouldPassValidationIfFieldLengthsAreCorrect() {
 		assertFalse(errors.hasErrors());
 	}
 	
+	/**
+	 * @see ProviderAttributeTypeValidator#validate(Object,Errors)
+	 */
+	@Test
+	public void validate_shouldFailValidationIfDatatypeClassnameIsEmpty() {
+		ProviderAttributeType type = new ProviderAttributeType();
+		type.setName("name");
```

Comment:
How is this related to the ticket?

Reply:
Thought I'd write the tests for stuff that wasn't covered for the validate method.

Reply:
No big deal though. You can leave it in. :)